Updated albert.md doc for ALBERT model (#27223)
huggingface/transformers: docs/source/en/model_doc/albert.md

- [`TFAlbertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/text-classification).

- [`FlaxAlbertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/flax/text-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/text_classification_flax.ipynb).
- Check the [Text classification task guide](../tasks/sequence_classification) on how to use the model.

<PipelineTag pipeline="token-classification"/>

- [`AlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/token-classification).

- [`TFAlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/token-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification-tf.ipynb).

> - [`AlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/token-classification).

Reviewer: This is also a duplicate ;)

Reviewer: Hi @ENate, would you mind removing the following duplicate line, then we can get your first PR merged!

```suggestion
```

- [`FlaxAlbertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/flax/text-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/text_classification_flax.ipynb).
- Check the [Text classification task guide](../tasks/sequence_classification) on how to use the model.

<PipelineTag pipeline="token-classification"/>

- [`AlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/token-classification).

- [`TFAlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/token-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification-tf.ipynb).

- [`AlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/token-classification).

> - [`TFAlbertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/token-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification-tf.ipynb).

Reviewer: Oops this is a duplicate of the `TFAlbertForTokenClassification` above!

```suggestion
```

Author: Thanks again :) I promise to do better next time :)

<PipelineTag pipeline="text-classification"/>

- A blog post on how to train AlBERT with Blurr for sequence classification can be done by adapting this [example](https://huggingface.co/blog/fastai).
- A blog post on how to use [Ray to tune hyperparameters](https://huggingface.co/blog/ray-tune) can be adapted for AlBERT.
- A notebook on how to [Fine-tune ALBERT for sentence-pair classification](https://github.com/NadirEM/nlp-notebooks/blob/master/Fine_tune_ALBERT_sentence_pair_classification.ipynb). 🌎
- A notebook on how to [Fine Tuning Transformer for MultiLabel Text Classification](https://github.com/abhimishra91/transformers-tutorials/blob/master/transformers_multi_label_classification.ipynb)

- Adapt this blog post on [Getting Started with Sentiment Analysis using Python](https://huggingface.co/blog/sentiment-analysis-python) for AlBERT.

> - [`AlBertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-classification).

Reviewer: The "B" should be lowercase (this typo occurs several times so I would double check to make sure they're all lowercase)

```suggestion
- [`AlbertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-classification).
```

Author: Ok sure :)

This model was contributed by [lysandre](https://huggingface.co/lysandre). This model jax version was contributed by
[kamalkraj](https://huggingface.co/kamalkraj). The original code can be found [here](https://github.com/google-research/ALBERT).

## Documentation Resources

The resources provided in the following sections consist of a list of official Hugging Face and community (indicated by 🌎) resources to help you get started with AlBERT. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.

<PipelineTag pipeline="text-classification"/>

> - A blog post on how to train AlBERT with Blurr for sequence classification can be done by adapting this [example](https://huggingface.co/blog/fastai).

Author: Thanks. I can make the changes again. First time contribution for me. Is that okay?

> - A blog post on how to train AlBERT with Blurr for sequence classification can be done by adapting this [example](https://huggingface.co/blog/fastai).

Reviewer: Yes of course, and welcome to the Hugging Face community! ❤️

> - A blog post on how to train AlBERT with Blurr for sequence classification can be done by adapting this [example](https://huggingface.co/blog/fastai).

Author: Thanks :)

- A blog post on how to use [Ray to tune hyperparameters](https://huggingface.co/blog/ray-tune) can be adapted for AlBERT.

> - A notebook on how to [Fine-tune ALBERT for sentence-pair classification](https://github.com/NadirEM/nlp-notebooks/blob/master/Fine_tune_ALBERT_sentence_pair_classification.ipynb)

Reviewer: ```suggestion
- A notebook on how to [Fine-tune ALBERT for sentence-pair classification](https://github.com/NadirEM/nlp-notebooks/blob/master/Fine_tune_ALBERT_sentence_pair_classification.ipynb). 🌎
```

> - A notebook on how to [Fine Tuning Transformer for MultiLabel Text Classification](https://github.com/abhimishra91/transformers-tutorials/blob/master/transformers_multi_label_classification.ipynb)

Reviewer: This can also be removed.

Author: So this example
```
- A notebook on how to [Fine-tune ALBERT for sentence-pair classification](https://github.com/NadirEM/nlp-notebooks/blob/master/Fine_tune_ALBERT_sentence_pair_classification.ipynb). 🌎
```
should also  be removed as well because it is not from the community?

Reviewer: That one is okay since it's linked from the [Community resources](https://huggingface.co/docs/transformers/community) doc.

> - Adapt this blog post on [Getting Started with Sentiment Analysis using Python](https://huggingface.co/blog/sentiment-analysis-python) for AlBERT.

Reviewer: Also remove

- [`AlBertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-classification).

> - [`TFAlBertForSequenceClassification`] can be done by adapting this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/text-classification).

Reviewer: ```suggestion
- [`TFAlBertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/text-classification).
```

- [`FlaxAlBertForSequenceClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/flax/text-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/text_classification_flax.ipynb).
- Check the [Text classification task guide](../tasks/sequence_classification) on how to use the model.

<PipelineTag pipeline="token-classification"/>

- [`AlBertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/token-classification) and can be implemented by adapting the example in this [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification.ipynb).

> - [`TFAlBertForTokenClassification`] is supported by adapting this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/token-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification-tf.ipynb).

Reviewer: ```suggestion
- [`TFAlBertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/token-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification-tf.ipynb).
```

- [`AlBertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/token-classification) and by adapting the [ example in this notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification.ipynb).
- [`TFAlBertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/token-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/token_classification-tf.ipynb).
- [`FlaxAlBertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/flax/token-classification).

> - [`FlaxAlBertForTokenClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/flax/token-classification).

Reviewer: Duplicate ;)

```suggestion
```

- [`AlbertForMultipleChoice`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/multiple-choice) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/multiple_choice.ipynb).
- [`TFAlbertForMultipleChoice`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/multiple-choice) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/multiple_choice-tf.ipynb).

- Check the  [Multiple choice task guide](../tasks/multiple_choice) on how to use the model.

⚗️ Optimization

- Blog posts on methods and tools [ for efficient training on a single GPU
](https://huggingface.co/docs/transformers/perf_train_gpu_one), [efficient Inference on a Single GPU](https://huggingface.co/docs/transformers/perf_infer_gpu_one) and [efficient Inference on CPU
](https://huggingface.co/docs/transformers/perf_infer_cpu) are also available.
- A blog post on how [Optimizing Transformers for GPUs with 🤗 Optimum](https://www.philschmid.de/optimizing-transformers-with-optimum-gpu) can be modified for ALBERT.

> - A blog post on [Optimizing Transformers with Hugging Face Optimum](https://huggingface.co/blog/optimum-inference) and [an example can be found here](https://www.philschmid.de/optimizing-transformers-with-optimum).

Reviewer: Let's also remove these sections since they aren't directly applicable to ALBERT

Author: But I saw that, in the following links,  training efficiency on CPUs is discussed for generic BERT-based models:
```
⚗️ Optimization

- A blog post [ for efficient training on a single GPU
](https://huggingface.co/docs/transformers/perf_train_gpu_one).
```
- while ALBERT is also referenced here as well:

```
- A blog post on [Optimizing Transformers with Hugging Face Optimum](https://huggingface.co/blog/optimum-inference).
```

Do I still remove the entire 'Optimization' section?

Reviewer: Yeah those are very generic and broad docs, so I think we should remove it entirely

⚡️ Inference

- A blog post on how to [Accelerate BERT inference with Hugging Face Transformers and AWS Inferentia](https://huggingface.co/blog/bert-inferentia-sagemaker) can be adapted for AlBERT.

> - A blog post on [Serverless Inference with Hugging Face's Transformers, DistilBERT and Amazon SageMaker](https://www.philschmid.de/sagemaker-serverless-huggingface-distilbert) can be adapted for AlBERT.

Reviewer: Let's also remove these

Author: Done

🚀 Deploy

- A blog post on how to [train ALBERT for natural language processing with TensorFlow on Amazon SageMaker](https://aws.amazon.com/blogs/machine-learning/train-albert-for-natural-language-processing-with-tensorflow-on-amazon-sagemaker/).

> - A post on how to [achieve 35% faster training with Hugging Face Deep Learning Containers on Amazon SageMaker](https://dataintegration.info/achieve-35-faster-training-with-hugging-face-deep-learning-containers-on-amazon-sagemaker).

Reviewer: Also remove since its not an official Hugging Face resource

> - Adapt the guide on this blog post on how to [Deploy BERT with Hugging Face Transformers, Amazon SageMaker and Terraform module](https://www.philschmid.de/terraform-huggingface-amazon-sagemaker) for ALBERT.

Reviewer: Also remove since its for DistilBERT
